packages/react-button/.storybook/preview.js

```diff
@@ -25,6 +29,16 @@ function loadStories() {
     generateStoriesFromExamples({ key, req, stories });
   });
 
+  // Wrap examples with ThemeProvider
+  for (let [key, story] of stories) {
+    Object.keys(story).forEach(exampleName => {
+      const example = story[exampleName];
+      if (typeof example === 'function') {
+        story[exampleName] = () => example();
+      }
+    });
+  }
+
```

Comment:
we should be able to remove this block here and in `react-next` too. could you do that and double check nothing breaks? thanks!